checker/check.ml

```diff
@@ -298,18 +298,27 @@ let name_clash_message dir mdir f =
 (* Dependency graph *)
 let depgraph = ref LibraryMap.empty
 
+let marshal_in_segment f ch =
+  try
+    let stop = input_binary_int ch in
+    let v = Analyze.instantiate (Analyze.parse_channel ch) in
```

Comment:
You are now using code from `checker/` in `coq`... I think such module should then be moved to coq.

Reply:
sorry, I got it wrong